### PR TITLE
chore(demo): name sources, add new sources, include a quality picker menu

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -77,22 +78,37 @@ <h3>Load a URL</h3>
...
-      <option value="https://s3.amazonaws.com/bob.jwplayer.com/~alex/121628/new_master.m3u8">ts Audio only</option>
-
-      <option value="https://akamai-axtest.akamaized.net/routes/lapd-v1-acceptance/www_c4/Manifest.m3u8">ts Live</option>
-      <option value="https://playertest.longtailvideo.com/adaptive/oceans_aes/oceans_aes.m3u8">ts Encrypted</option>
-      <option value="https://bitmovin-a.akamaihd.net/content/sintel/hls/playlist.m3u8">ts with audio/subs and a 4k rendtion</option>
+      <option value="https://devstreaming-cdn.apple.com/videos/streaming/examples/img_bipbop_adv_example_ts/master.m3u8">Advanced Bipbop - ts and captions muxed</option>
+      <option value="https://devstreaming-cdn.apple.com/videos/streaming/examples/img_bipbop_adv_example_fmp4/master.m3u8">Advanced Bipbop - FMP4 and captions muxed</option>
+      <option value="https://devstreaming-cdn.apple.com/videos/streaming/examples/bipbop_adv_example_hevc/master.m3u8">Advanced Bipbop - FMP4 hevc, demuxed</option>
+      <option value="https://storage.googleapis.com/shaka-demo-assets/angel-one-hls/hls.m3u8">Angel One - FMP4 unmuxed, many audio/captions</option>
```
Comment (gesinger) [inline index.html:86]:
```suggestion
      <option value="https://storage.googleapis.com/shaka-demo-assets/angel-one-hls/hls.m3u8">Angel One - FMP4 demuxed, many audio/captions</option>
```

File: index.html [hunk 2]
```diff
@@ -77,22 +78,37 @@ <h3>Load a URL</h3>
...
+      <option value="https://bitdash-a.akamaihd.net/content/MI201109210084_1/m3u8s-fmp4/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8">Parkour - FMP4 unmuxed</option>
```
Comment (gesinger) [inline index.html:87]:
```suggestion
      <option value="https://bitdash-a.akamaihd.net/content/MI201109210084_1/m3u8s-fmp4/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8">Parkour - FMP4 demuxed</option>
```

File: index.html [hunk 3]
```diff
@@ -77,22 +78,37 @@ <h3>Load a URL</h3>
...
+      <option value="https://playertest.longtailvideo.com/adaptive/oceans_aes/oceans_aes.m3u8">Disney's Oceans trailer - HLSe, ts Encrypted</option>
+      <option value="https://bitmovin-a.akamaihd.net/content/sintel/hls/playlist.m3u8">Sintel - ts with audio/subs and a 4k rendtion</option>
+      <option value="https://storage.googleapis.com/shaka-demo-assets/bbb-dark-truths-hls/hls.m3u8">BBB-CMIF: Big Buck Bunny Dark Truths - demuxed, fmp4</option>
     </optgroup>
     <optgroup label="dash">
-      <option value="https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd">unmuxed audio/video</option>
-      <option value="https://vm2.dashif.org/livesim/mup_30/testpic_2s/Manifest.mpd">Live</option>
-      <option value="https://dash.akamaized.net/dash264/TestCases/10a/1/iis_forest_short_poem_multi_lang_480p_single_adapt_aaclc_sidx.mpd">Sidx unmuxed, 2 audio</option>
+      <option value="https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd">Big Buck Bunny - unmuxed audio/video, includes 4K, burns in frame, pts, resolution, bitrate values</option>
```
Comment (gesinger) [inline index.html:95]:
```suggestion
      <option value="https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd">Big Buck Bunny - demuxed audio/video, includes 4K, burns in frame, pts, resolution, bitrate values</option>
```

File: index.html [hunk 4]
```diff
@@ -77,22 +78,37 @@ <h3>Load a URL</h3>
   <h3>Load a Source</h3>
   <select id=load-source>
     <optgroup label="hls">
-      <option value="https://d2zihajmogu5jn.cloudfront.net/bipbop-advanced/bipbop_16x9_variant.m3u8">Muxed TS with 1 alt Audio, 5 captions</option>
+      <option value="https://d2zihajmogu5jn.cloudfront.net/bipbop-advanced/bipbop_16x9_variant.m3u8">Bipbop - Muxed TS with 1 alt Audio, 5 captions</option>
```
Comment (brandonocasey) [inline index.html:81]: What do you think of loading all of these sources from a json file?

File: index.html [hunk 5]
```diff
@@ -77,22 +78,37 @@ <h3>Load a URL</h3>
...
     </optgroup>
     <optgroup label="dash">
-      <option value="https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd">unmuxed audio/video</option>
-      <option value="https://vm2.dashif.org/livesim/mup_30/testpic_2s/Manifest.mpd">Live</option>
-      <option value="https://dash.akamaized.net/dash264/TestCases/10a/1/iis_forest_short_poem_multi_lang_480p_single_adapt_aaclc_sidx.mpd">Sidx unmuxed, 2 audio</option>
+      <option value="https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd">Big Buck Bunny - unmuxed audio/video, includes 4K, burns in frame, pts, resolution, bitrate values</option>
+      <option value="https://storage.googleapis.com/shaka-demo-assets/angel-one/dash.mpd">Angel One - fmp4, webm, subs, alternate audio tracks</option>
+      <option value="https://storage.googleapis.com/shaka-demo-assets/bbb-dark-truths/dash.mpd">BBB-CMIF: Big Buck Bunny Dark Truths - demuxed, fmp4</option>
+      <option value="https://dash.akamaized.net/dash264/TestCases/10a/1/iis_forest_short_poem_multi_lang_480p_single_adapt_aaclc_sidx.mpd">SIDX unmuxed, 2 audio</option>
```
Comment (gesinger) [inline index.html:98]:
```suggestion
value="https://dash.akamaized.net/dash264/TestCases/10a/1/iis_forest_short_poem_multi_lang_480p_single_adapt_aaclc_sidx.mpd">SIDX demuxed, 2 audio</option>
```

File: index.html [hunk 6]
```diff
@@ -77,22 +78,37 @@ <h3>Load a URL</h3>
...
+      <option value="https://download.tsi.telecom-paristech.fr/gpac/DASH_CONFORMANCE/TelecomParisTech/mp4-onDemand/mp4-onDemand-mpd-AV.mpd">SIDX bipbop-like</option>
+      <option value="https://yt-dash-mse-test.commondatastorage.googleapis.com/media/car-20120827-manifest.mpd">Google self-driving car - SIDX</option>
+      <option value="https://d2zihajmogu5jn.cloudfront.net/sintel_dash/sintel_vod.mpd">Sintel - single rendition</option>
+    </optgroup>
+    <optgroup label="live">
+      <option value="https://akamai-axtest.akamaized.net/routes/lapd-v1-acceptance/www_c4/Manifest.m3u8">HLS - Live - Axinom live stream, may not always be available</option>
```
Comment (gesinger) [inline index.html:104]: If it's not always available, I think we may be better off not including it.

File: scripts/index-demo-page.js [hunk 7]
```diff
@@ -205,6 +206,12 @@
         stateEls.partial.dispatchEvent(newEvent('change'));
 
         player = window.player = window.videojs(videoEl, {
+          plugins: {
+            httpSourceSelector:
+            {
```
Comment (gesinger) [inline scripts/index-demo-page.js:211]:
```suggestion
            httpSourceSelector: {
```

File: index.html (hunk 4 above)
Comment (gkatsev) [reply index.html:81]: Not a bad idea, I can do it as part of another PR, would like to get this in so it's easier for people to test.

File: index.html (hunk 6 above)
Comment (gkatsev) [reply index.html:104]: It's still helpful when it's up but want to let people know that if it 404ing it's expected.

**Pull request merged**